From 6a1782fa64511901cfe07d58fc1fc86c43e39a96 Mon Sep 17 00:00:00 2001
From: Bharat B Panda <bharat.b.panda@intel.com>
Date: Thu, 8 Aug 2024 12:59:33 +0000
Subject: [PATCH] Fix wifi connection failure

Changes made to disable PASN (Pre-Association Security Negotiation),
as we do not support respective sub-feature in iwlwifi driver.

Test Done:
1. Flash the image BM and CiV
2. After boot, connect to an AP
3. Connection Successfull

Tracked-On: OAM-123250
Signed-off-by: Bharat B Panda <bharat.b.panda@intel.com>
---
 wpa_supplicant/android.config | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/wpa_supplicant/android.config b/wpa_supplicant/android.config
index 4cc38084..4c8fd230 100644
--- a/wpa_supplicant/android.config
+++ b/wpa_supplicant/android.config
@@ -542,7 +542,7 @@ CONFIG_DPP2=y
 CONFIG_SAE=y
 
 # PASN
-CONFIG_PASN=y
+#CONFIG_PASN=y
 
 # WPA3-Enterprise (SuiteB-192)
 CONFIG_SUITEB=y
-- 
2.34.1

